src/main/java/com/kodilla/ecommercee/mapper/ItemMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.mapper;
+
+import com.kodilla.ecommercee.domain.order.Item;
+import com.kodilla.ecommercee.domain.order.Order;
+import com.kodilla.ecommercee.domain.product.Product;
+import com.kodilla.ecommercee.domain.product.ProductDto;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class ItemMapper {
```

Comment:
to nie jest mapper. mapper w tym wypadku starały się zamienić obiekty typu `Item` na `ItemDto` lub coś w tym stylu. ta klasa służy bardziej jako assembler.